core/tests/test_utils.py

```diff
@@ -143,16 +148,43 @@ class TestBase(unittest.TestCase):
     }
 
     def _get_unicode_test_string(self, suffix):
+        """Returns a string for unicode compatibility test.
+
+        Args:
+            suffix: string. Appended to unicode string.
+
+        Returns:
+            string. Unicode test string.
+        """
         return '%s%s' % (self.UNICODE_TEST_STRING, suffix)
 
     def setUp(self):
+        """Sets up environment for Test Base
+            (implemented in inherited class).
```

Comment:
Ditto, move what you can to the previous line. Same for all docstrings.

Reply:
corrected throughout file

Reply:
corrected throughout file